packages/@okta/vuepress-theme-prose/util/attribution/attribution.js

```diff
@@ -0,0 +1,269 @@
+import queryString from "query-string";
+import moment from "moment-timezone";
+import { deleteCookie, getCookie, setCookie } from "./cookies";
+
+const acceptedParams = [
+  "utm_campaign",
+  "utm_content",
+  "utm_medium",
+  "utm_source",
+  "utm_term"
+];
+
+const allParams = [
+  "utm_campaign",
+  "utm_content",
+  "utm_date",
+  "utm_medium",
+  "utm_page",
+  "utm_source",
+  "utm_term"
+];
+
+let isAttached = false;
+
+function filterParams(params) {
+  if (typeof params == "string") {
+    params = queryString.parse(params);
+
+    params = Object.keys(params)
+      .filter(key => acceptedParams.indexOf(key) !== -1)
+      .reduce((object, key) => {
+        object[key] = params[key];
+        return object;
+      }, {});
+  }
+
+  params = Object.keys(params)
+    .filter(key => allParams.indexOf(key) !== -1)
+    .reduce((object, key) => {
+      object[key] = params[key];
+      return object;
+    }, {});
+
+  params = Object.assign(
+    {
+      utm_page: location.pathname,
+      utm_date: moment(moment.utc())
+        .tz("America/Los_Angeles")
+        .format("MM/DD/YYYY")
+    },
+    params
+  );
+
+  return params;
+}
+
+/**
+ * onAttach function.
+ *
+ * @access public
+ * @return void
+ */
+function onAttach() {
+  if (isAttached == false) {
```

Comment:
nit: strong preference against `==`
```suggestion
  if (isAttached === false) {
```